### DISCUSSION TITLE
I've been out of commission for a couple of weeks

### DISCUSSION THREAD

**Discussion by SteveIreland** [opensourcepos/opensourcepos#3674] [Team Posts]
I had to have another carotid artery surgery about 10 days ago  (an interesting experience during a pandemic) but I'm rapidly recovering.  Hopefully I haven't been left too far behind and can get back into the OSPOS game!

**Comment by odiea** [comment]
Hope you get better.

**Comment by WebShells** [comment]
Hello @SteveIreland hope you recover very soon, sad news... Get well soon!

**Comment by jekkos** [comment]
Get better soon!l. Take care and I hope to see you back here indeed!

**Comment by objecttothis** [comment]
@SteveIreland sorry I just saw this.  How are you feeling?
